qa-theme/SnowFlat/qa-styles-rtl.css

```diff
@@ -310,8 +311,7 @@ h1 {
 }
 
 .qa-q-view-avatar {
-	right: auto;
-	left: 20px;
```

Comment:
Any reason dropping the 20px here rather than keeping it?

Reply:
I'd answer with the opposite question: what was the reason to add it there in the first place?